packages/odyssey-react-mui/src/test-selectors/interpolateString.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export const isRegExpString = (string: string) => /^\/*(.+)\/$/.test(string);
+
+export const interpolateString = (
```

Comment:
Could you add some comments here clarifying what this should be used for? What risks if any, are we exposing ourselves to with using eval like this?

Reply:
I actually removed the functionality where this would be used, and I'm not sure I need to add it back at this point.

I'll put a note in here. The `eval` is safe because it only `eval`s what someone passes to Odyssey from their unit test.

I might remove this function entirely in a future, so I marked it as `@deprecated`.